### ISSUE TITLE
RLN: Update identity_secret generation

### ISSUE THREAD

**Issue by rymnc** [vacp2p/zerokit#81]
To make the credentials generated by RLN more compatible with Semaphore, we need to update the secret generation to include generation of the `identity_trapdoor` and `identity_nullifier` as mentioned in Semaphore's docs - https://semaphore.appliedzkp.org/docs/technical-reference/circuits

This will allow re-using the same credentials for Semaphore groups and RLN

cc: @s1fr0 @oskarth

**Comment by rymnc** [comment]
cc: @AtHeartEngineer from the PSE team

**Comment by oskarth** [comment]
This would require a spec change right? If so we should have an issue and discussion/decision in RFC repo

**Comment by rymnc** [comment]
I don't think this requires a spec change. The previous implementation was not according to spec, and now it is :)
